README.md

```diff
@@ -8,7 +8,7 @@ with Git](https://source.android.com/source/version-control.html).
 
 To initialize your local repository using the Maru OS trees, use a command like this:
 
-    repo init -u git://github.com/maruos/manifest.git -b maru-0.6
+    repo init -u git://github.com/maruos/manifest.git -b maru-0.7
```

Comment:
A new PR for this modification has been merged. Maybe you should rebase to the newest code base.